usage.md

```diff
@@ -55,3 +79,28 @@ We also considered the following way to override the automatic detection, but cu
   <p>You can also manually disable paragraphs using the data-content-paragraphs="false" config.</p>
 </div>
 ```
+
+## renuo-cms with rails
```

Comment:
1. renuo-cms: when we use it in text, we call it Renuo CMS
2. this is a title, so it should be capitalized